target_priority.opy

```diff
@@ -0,0 +1,53 @@
+#!mainFile "main.opy"
+
+# Threat. Shows the potential danger of the enemy. This value is only for players, i.e bots always have 0 threat.
```

Comment:
bots can also be on enemy team, so it should be "allies always have 0 threat"

Reply:
if you mean ai bots, then eh, i think it's understandable enough for anyone reading the code

if you mean dummy bots, then are we doing bots vs bots? 👀

Reply:
yup, how else are we going to debug :p